modules/_node-scripts/tsc/index.mjs

```diff
@@ -18,11 +18,11 @@ import getFileProjectDir from '../util/getFileProjectDir.mjs';
 import getGitModifiedFiles from '../util/getGitModifiedFiles.mjs';
 import getNamedArguments from '../util/getNamedArguments.mjs';
 
-export default async function main() {
-	const {modifiedSince} = getNamedArguments({
-		modifiedSince: '--modified-since',
-	});
+const {modifiedSince} = getNamedArguments({
```

Comment:
I would move this logic inside `main()` and use it if no argument is provided. I know it works in both ways, but using a top level const that has been computed by a function as a default argument feels a little magical to me. :grimacing: